docs/README.md

```diff
@@ -7,6 +7,22 @@ _Modern Firefox Telemetry for mobile platforms_
 `Glean` is a modern approach for a mobile Telemetry library.
 It started out as an [android-components library](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean), written purely in Kotlin to be used on Android.
 
-`Glean` is an experiment to transform the concepts behind `Glean` into a cross-platform library based on a low-level [Rust](https://www.rust-lang.org/) library and the necessary platform integrations for Android and iOS.
+`Glean` is a work-in-progress to transform the concepts behind `Glean` into a cross-platform library based on a low-level [Rust](https://www.rust-lang.org/) library and the necessary platform integrations for Android and iOS (and beyond...).
+
+To contact us you can:
+- Find us on the Mozilla Slack in *#glean*, on [Mozilla IRC](https://wiki.mozilla.org/IRC) in *#telemetry*.
+- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data%20Platform%20and%20Tools&component=Glean%3A%20SDK).
+- Send an email to *glean-team@mozilla.com*.
+- The Glean team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*
+
+**Note: The cross-platform `Glean` project is in development. It's not finished,
```

Comment:
nit: I would move this before the contact section, so that it's immediately visible.